RELEASES.md

```diff
@@ -256,15 +256,6 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
 2. Perform the steps from `I.3` above but use a `~stableppaX` suffix instead of `~rcX` in the version name, and upload to `ppa:ua-client/stable` instead of staging.
 3. Bring in any changes that were made to the release branch into `main` via PR (e.g. Changelog edits).
 
-## Ubuntu PRO Release Process
-
-Below is the procedure used to release ubuntu-advantage-tools to Ubuntu PRO images:
-
- 1. [Open Daily PPA copy-package operation](https://code.launchpad.net/~ua-client/+archive/ubuntu/daily/+copy-packages)
- 2. Check Xenial, Bionic, Focal, Hirsute, Impish packages
- 3. Select Destination PPA: UA Client Premium [~ua-client/ubuntu/staging]
- 4. Select Destination series: The same series
- 5. Copy options: "Copy existing binaries"
- 6. Click Copy packages
- 7. Notify Pro Image creators about expected Premium PPA version (patviafore/powersj)
- 8. Once new PRO AMIs are publicly available run `./tools/refresh-aws-pro-ids` to update AMIs we test during CI runs
+## Cloud Images Update
+
+After the release process is finished, if there is a need to push a new cloud image with the new version of UA, CPC must be informed. They will be responsible to update the images using the package from the pockets it was released to (whether it is the `stable` PPA or the`-updates` pocket).
```

Comment:
I think we should always ping CPC with a new release. But now I have a another question regarding the release. When should we upload the package to `stable` ? If we do it after the package hits `updates`, then there is no need anymore for using the package on stable and if we upload it to stable after the package moves to `proposed`, errors can be found on the package during the testing phase invalidating that package as `stable`.

So I don't know if we should keep that ppa anymore. What do you think @CalvoM @renanrodrigo  @blackboxsw @orndorffgrant ?

Reply:
Updated according to post-standup discussion